<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Log Viewer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QListWidget" name="listWidget">
      <property name="selectionMode">
       <enum>QAbstractItemView::MultiSelection</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>460</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuApplication">
    <property name="title">
     <string>Application</string>
    </property>
    <addaction name="actionSettings"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionRefresh"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="actionCloseAll"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuPlot">
    <property name="title">
     <string>Plot</string>
    </property>
    <addaction name="actionPlot"/>
    <addaction name="actionSpectrum"/>
    <addaction name="separator"/>
    <addaction name="actionSelectAll"/>
    <addaction name="actionSelectClear"/>
    <addaction name="separator"/>
    <addaction name="actionAssign"/>
    <addaction name="actionClear"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuPlot"/>
   <addaction name="menuWindow"/>
   <addaction name="menuApplication"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="LogViewerToolbar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="toolTip">
    <string>Open application settings</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
   <property name="toolTip">
    <string>Refresh file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="toolTip">
    <string>Close current file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionCloseAll">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Close All</string>
   </property>
   <property name="toolTip">
    <string>Close all opened windows with plots</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+W</string>
   </property>
  </action>
  <action name="actionPlot">
   <property name="text">
    <string>Generate Plot</string>
   </property>
   <property name="toolTip">
    <string>Generate a plot</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionSpectrum">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Spectrum</string>
   </property>
   <property name="toolTip">
    <string>Plot spectrums instead of values</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionAssign">
   <property name="text">
    <string>Assign To Plot</string>
   </property>
   <property name="toolTip">
    <string>Assign to a single plot</string>
   </property>
   <property name="shortcut">
    <string>+</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>Clear Assignment</string>
   </property>
   <property name="toolTip">
    <string>Clear assignment</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionSelectAll">
   <property name="text">
    <string>Select All</string>
   </property>
   <property name="toolTip">
    <string>Select all items</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionSelectClear">
   <property name="text">
    <string>Clear Selection</string>
   </property>
   <property name="toolTip">
    <string>Clear selection</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
   <property name="toolTip">
    <string>About application</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LogViewerToolbar</class>
   <extends>QToolBar</extends>
   <header>toolbar</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
